libc/sem.cc

```diff
@@ -76,3 +105,81 @@ int sem_trywait(sem_t* s)
         return libc_error(EAGAIN);
     return 0;
 }
+
+//TODO: when are these initialized?
```

Comment:
These would be initialized as part of the kernel ELF init which is very early on (look at premain() and inittab in loader.cc).

Reply:
Ah, I figured it must be somewhere!